### ISSUE TITLE
Script creation does not save script

### ISSUE THREAD

**Issue by simonastoyanova** [kingsdigitallab/crossreads#22]
I tried to add the Greek alphabet under Greek 1 in Scripts. Saved and went to the Annotator tab, tried to annotate an image with Greek 1 and it still only had the 5 test letters in there. Looks like a bug but do let me know if I'm missing a step.

**Comment by geoffroy-noel-ddh** [comment]
(The definition screen was broken, due to Git API library upgrade. That's now fixed.)

I can't reproduce the issue. Here's what I've tried, step by step:
1. [Go to annotator, select an image and a box](https://kingsdigitallab.github.io/crossreads/annotator.html?obj=http://sicily.classics.ox.ac.uk/inscription/ISic000001&img=ISic000001.jpg&ann=7e06475e-3de9-4bfe-b8bd-71e383e399dd&scr=latin-1)
2. Click the Definitions tab
3. Select Greek 1, add a new letter (e.g. H)
4. Click the green save button in the Definitions tab
5. Click the Annotator tab (now the box should still be selected)
6. Open the allograph drop-down: the new letter should be visible

Maybe it's because you didn't click Save? I'll improve the definition so it can auto-save. It is confusing as the Annotator tab auto-saves but not the Definitions tab.

Let me know if you can still reproduce the bug.

**Comment by simonastoyanova** [comment]
OK, I think the problem is actually different but I don't understand why it happens. I tried adding H and then I, and they are both saved and visible once I switch back to the Annotator. All fine. I used this same process previously, too, so apart from adding these one at a time for testing, I haven't done anything different. 

However, when I try to add a Greek letter to this test set in Greek 1, it doesn't save it. I tried with several letters at once, I tried one at a time. Is it possible that not saving these letters is related to the change of alphabet? I'm only switching my keyboard, it's all Unicode and it's the basic Greek alphabet with no diacritics or odd symbols. I also tried deleting all the current testing letters in Greek 1 and adding the entire Greek alphabet, in case the mixing of alphabets in one Script was the issue, but that also doesn't save the changes and reverts back to the test set of Greek 1.
